.github/workflows/test-jobs/30-CT_SUITE.yaml

```diff
@@ -0,0 +1,77 @@
+  # vim:sw=2:et:
+  ct-$(CT_SUITE):
+    needs: [checks]
+    # https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#contexts
+    name: ct-$(CT_SUITE)
+    runs-on: ubuntu-18.04
+    steps:
+      - name: CHECKOUT REPOSITORY
+        uses: actions/checkout@v2
+      # https://github.com/marketplace/actions/setup-elixir
+      - name: CONFIGURE OTP & ELIXIR
+        uses: actions/setup-elixir@v1
+        with:
+          otp-version: $(ERLANG_VERSION)
+          # https://github.com/elixir-lang/elixir/releases
+          elixir-version: $(ELIXIR_VERSION)
+      - name: CACHE DEPS
+        uses: actions/cache@v1
+        with:
+          path: deps
+          key: otp-$(ERLANG_VERSION)_git-${{ github.sha }}_deps
+      - name: RUN TESTS
+        run: |
+          branch_or_tag_name=${GITHUB_REF#refs/*/}
+          ! test -d ebin || touch ebin/*
+          export BASE_RMQ_REF=$(base_rmq_ref)
+          export ERLANG_VERSION=$(ERLANG_VERSION)
+          export ELIXIR_VERSION=$(ELIXIR_VERSION)
+          make ct-$(CT_SUITE) \
+            base_rmq_ref=$(base_rmq_ref) \
+            current_rmq_ref=$branch_or_tag_name \
+            FULL= \
+            FAIL_FAST=1 \
+            SKIP_AS_ERROR=1 \
+            CT_OPTS="-ct_hooks honeycomb_cth '[{directory,\"$PWD/honeycomb\"}]'"
+      - name: HONEYCOMB
+        run: |
+          for f in honeycomb/*; do
+            curl -H 'X-Honeycomb-Team: ${{ secrets.HONEYCOMB_TEAM }}' \
```

Comment:
This should receive the same return code validation as was added to rabbitmq-cli, and probably also move to the tail of the steps, so that if it fails, it does not affect calls to `success()` in the `if:` of the mixed versions tests.